### PR TITLE
Add isARepeat

### LINKED ISSUE #316: add isARepeat method to NSEvent?
This seems like it is probably an oversight; I think this should be implemented alongside `keyCode()`, `characters()`, etc?

### PR DESCRIPTION
**Pull request by cmyr**
closes #316

---
This change is [Reviewable](https://reviewable.io/reviews/servo/core-foundation-rs/317)

### CODE REVIEW

File: cocoa/src/appkit.rs [hunk 1]
```diff
@@ -2354,6 +2354,7 @@ pub trait NSEvent: Sized {
     unsafe fn characters(self) -> id /* (NSString *) */;
     unsafe fn charactersIgnoringModifiers(self) -> id /* (NSString *) */;
     unsafe fn keyCode(self) -> libc::c_ushort;
+    unsafe fn isARepeat(self) -> bool;
```
Comment (jdm) [inline cocoa/src/appkit.rs]: Use `BOOL` instead.

File: cocoa/src/appkit.rs [hunk 2]
```diff
@@ -2602,6 +2603,10 @@ impl NSEvent for id {
         msg_send![self, keyCode]
     }
 
+    unsafe fn isARepeat(self) -> bool {
```
Comment (jdm) [inline cocoa/src/appkit.rs]: Use BOOL instead.

**Comment by cmyr** [comment]
@jdm updated.

**Comment by jdm** [comment]
@bors-servo r+
Thanks!

**Pull request merged**